example/spec/parallel/ingestGranule/ingestGranuleQueueSpec.js

```diff
@@ -532,14 +532,16 @@ describe('The S3 Ingest Granules workflow', () => {
         if (beforeAllError) fail(beforeAllError);
         if (subTestSetupError) fail(subTestSetupError);
       });
-
-      it('publishes the granule metadata to CMR', () => {
+      it('pretends to work', () => {
```

Comment:
This looks like it should be removed

Reply:
https://github.com/nasa/cumulus/pull/3544/files#r1447919653.      Interesting - if we commented out things such that before/after don't trigger, that implies downstream tests are relying on upstream test block behavior... which sounds like a refactor is in order, but *not* for this PR.       Please ignore the comments (resolved) that this should be removed if that's the case.